MechJeb2.sln

```diff
@@ -0,0 +1,27 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.25420.1
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MechJeb2", "MechJeb2\MechJeb2.csproj", "{9FC90AE6-C2E4-47F1-A6D1-DB4731A40BE3}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MechJeb2Tests", "MechJeb2Tests\MechJeb2Tests.csproj", "{C17BF82C-F906-4A92-8B9A-F8103CDB565C}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{9FC90AE6-C2E4-47F1-A6D1-DB4731A40BE3}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{9FC90AE6-C2E4-47F1-A6D1-DB4731A40BE3}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{9FC90AE6-C2E4-47F1-A6D1-DB4731A40BE3}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{9FC90AE6-C2E4-47F1-A6D1-DB4731A40BE3}.Release|Any CPU.Build.0 = Release|Any CPU
+		{C17BF82C-F906-4A92-8B9A-F8103CDB565C}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{C17BF82C-F906-4A92-8B9A-F8103CDB565C}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{C17BF82C-F906-4A92-8B9A-F8103CDB565C}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{C17BF82C-F906-4A92-8B9A-F8103CDB565C}.Release|Any CPU.Build.0 = Release|Any CPU
```

Comment:
there's a bunch of GUIDs here that are copypasta from B9PartSwitch{Tests} my guess is they're wrong.

Reply:
(i cooked this up in vim -- not Visual Studio or MonoDevelop)

Reply:
I have no idea how these are generated